lib/Ohmbrewer_Temperature_Sensor.h

```diff
@@ -178,6 +184,11 @@ namespace Ohmbrewer {
              */
             Probe* _probe;
 
+            /**
```

Comment:
Rather than duplicating the value of the display setting from the singleton settings object in every TemperatureSensor instance, write a Temperature#toStr(bool units) method that calls either Temperature#toStrC() or Temperature#toStrF() based on the units parameter (defaulting to celsius).

That should save on the number of writes to memory vs. what you've got going on now and avoid the penalty for setting that on every TemperatureSensor every time we refresh the display.